docs/contributing/Target Framework Strategy.md

```diff
@@ -110,3 +110,22 @@ When the .NET SDK RTMs and Roslyn adopts it all occurrences of `$(NetRoslynNext)
 
 **DO NOT** include both `$(NetRoslyn)` and `$(NetRoslynNext)` in the same project unless there is a very specific reason that both tests are adding value. The most common case is that the runtime has changed behavior and we simply need to update our baselines to match it. Adding extra TFMs for this just increases test time for very little gain.
 
+## Checklist for updating TFMs (once a year)
```

Comment:
One other item to check is that you don't regress the number of tests that are running in CI. Our logic for finding tests can include looking at TFM. In the past I messed this up and stopped running certain unit test assemblies.